tests/Stats/Persister/RedisPersisterTest.php

```diff
@@ -0,0 +1,279 @@
+<?php
+
+namespace App\Tests\Stats\Persister;
+
+use App\Stats\Persister\RedisPersister;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Predis\Client as Redis;
+
+class RedisPersisterTest extends TestCase
+{
+    /** @var Redis|MockObject */
+    private $redis;
+
+    protected function setUp(): void
+    {
+        $this->redis = $this->getMockBuilder(Redis::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+    }
+
+    public function testIncrementTotalAccess(): void
+    {
+        $now = new \DateTime();
+
+        $this->redis
+            ->expects($this->at(0))
```

Comment:
I don't like the `at` predicate, as it bound test code to implementation one. As I saw it is been used in other tests, I've followed what was already done.

Reply:
Fixed with `withConsecutive...` in order to get rid of warnings after rabasing to PHPUnit 9 upgrade.